src/main/java/web/pages/Page.java

```diff
@@ -0,0 +1,36 @@
+package web.pages;
+
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public class Page {
```

Comment:
Each Page should have a class.
So divide the locators/methods inside this to 2 class files under the package flipkart: 
HomePage
SearchPage